test/JDBC/input/BABEL_OBJECT_NAME-vu-verify.mix

```diff
@@ -0,0 +1,132 @@
+-- test table, trigger, procedure, function
```

Comment:
have we matched the expected output against SQL server?

Reply:
yes